modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,57 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Optional: Enabling monitoring of event listeners for Triggers for user-defined projects
```

Comment:
@michaelryanpeter You mentioned using "Optional" in today's OpenShift docs team meeting. Do you have any suggestions or comments on this instance?
https://docs.google.com/document/d/1CH-K3UAuAGbeqp0W-9AHH2v7OrCQawsxt4fiDKLj86Q/edit?disco=AAAAakoYDio

Reply:
- Using "optional" in titles is a complex issue that has come up for discussion recently.
- Red Hat seems to prefer not using Optional in titles this way. 
- If this optional module is the exception to a series of required tasks in an assembly, let's find a better way to make that obvious.  

```suggestion
= Enabling monitoring of event listeners for Triggers for user-defined projects
```

Reply:
```suggestion
As a cluster administrator, to gather event listener metrics for Triggers in a user-defined project and display them in the web console, you can create a service monitor for each event listener. On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`.
```

Reply:
The consensus from the OpenShift documentation team is that using "optional" in a heading creates problems for content reuse. As @rolfedh mentioned, explaining the conditional context of the procedure in the abstract is considered a best practice.